src/ctelnet.cpp

```diff
@@ -946,7 +946,7 @@ void cTelnet::processTelnetCommand(const string& command)
 
                 int newVersion = version.toInt();
                 if (mpHost->mServerGUI_Package_version != newVersion) {
-                    QString _smsg = tr("<The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'>")
+                    QString _smsg = tr("[INFO] The server wants to upgrade the GUI to new version '%1'. Uninstalling old version '%2'")
```

Comment:
You need a ` - ` between the `[ INFO ]` and the start of the message text - it is used programatically (in `cTelnet::postMessage(...)`) to mark the point (well the space after it) where following lines are indented in messages of more than one line (unless they get wrapped by `TBuffer` code for being overlong anyhow) - as a rule of thumb I had tried to work to a total message line length of no more than `80` characters but of course that gets blown out of the water when parametric string replacement happens OR the lines get translated...

Reply:
Also, I've tried to use `[ INFO ] - ` messages at the start/middle of a process to be followed by an `[  OK  ] - ` one at the end - so that if I was coding it, there would be a confirmation that the new GUI was downloaded and installed correctly (or an `[ ERROR ] - ` if it had not)...

Reply:
I was actually reviewing mudlet.ts for the "correct" way to write [INFO] text. The first example I found was from ircformatter.cpp and did not have any additional space or minus characters. Looking at it again, it was also the ONLY example without the added minus. OK, will rectify.
OK and ERROR will test, but may not add it, if it seems to complicated and is not yet established at all for this case.